README.md

```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
 
 With this you should be able to run the *Cardiac MRI segmentation-checkpoint.ipynb*.
 
-## **Container**
\ No newline at end of file
+## **Container**
+
+### Requirements
+The project can be run through a Docker container. Docker has to be installed on computer, for this to work.
```

Comment:
The project can be run with a Docker container. Docker has to be installed on this computer, for this to work.

Reply:
changed to "run in a Docker container"